search-app/src/cmr/search/api/request_context_user_augmenter.clj

```diff
@@ -38,12 +41,11 @@
   (let [{:keys [token]} context]
     (let [sids (cache/get-value (cache/context->cache context token-sid-cache-name)
                                 token
-                                #(or (sids-retriever/get-sids context token)
-                                     {:sids (acl/context->sids context)}))
-          token-guid (:guid sids)
```

Comment:
I'm fine with you merging this change in without a solution for our workload driver to use the same token. We can create a separate ticket for that.